### PR TITLE
Namespace experiments

### PR DESCRIPTION
**Pull request by gcotelli**
Attempt to replace constants classes with namespaces

### CODE REVIEW

File: source/RenoirSt-Tests/CssConstantsTest.class.st [hunk 1]
```diff
@@ -4,200 +4,203 @@ A CssConstantsTest is a test class for testing the behavior of CssConstants
 Class {
 	#name : #CssConstantsTest,
 	#superclass : #TestCase,
+	#pools : [
+		'RenoirSt'
```
Comment (fortizpenaloza) [inline source/RenoirSt-Tests/CssConstantsTest.class.st]: So declaring it as a shared pool will make us to add it to every web view class (or just to a common super class)?

File: source/RenoirSt-Tests/CssMediaQueryRuleTest.class.st [hunk 2]
```diff
@@ -21,13 +25,16 @@ CssMediaQueryRuleTest >> testPrintStringOfSimpleMediaQuery [
...
 
 	| styleSheet mediaQuery |
+
 	styleSheet := CascadingStyleSheet withAll: #().
 	mediaQuery := CssMediaQueryRule
-		ofType: CssMediaQueryConstants screen
+		ofType: CssMediaQueryConstants >> #screen
 		conforming:
 			{(CssMediaQueryExpression forFeatureNamed: 'color').
 			(CssMediaQueryExpression forFeatureNamed: 'orientation' withValue: 'landscape')}
```
Comment (mtabacman) [inline source/RenoirSt-Tests/CssMediaQueryRuleTest.class.st]: shouldn't this string use the CssMediaQueryConstants?

File: source/RenoirSt/RenoirSt.class.st [hunk 3]
```diff
@@ -0,0 +1,629 @@
...
+		bind: #square to: 'square';
+		bind: #upperAlpha to: 'upper-alpha';
+		bind: #upperLatin to: 'upper-latin';
+		bind: #upperRoman to: 'upper-roman'
+]
+
+{ #category : #'private - initialization' }
+RenoirSt class >> initializeMediaQueryConstants [
+
+	<ignoreForCoverage>
```
Comment (mtabacman) [inline source/RenoirSt/RenoirSt.class.st]: Shouldn't there be more examples on how to access this namespace? Is is included as a block parameter in a utility method?

File: source/RenoirSt-Tests/CssBoxShadowTest.class.st [hunk 4]
```diff
@@ -1,23 +1,26 @@
...
 	insetShadow := CssBoxShadow horizontalOffset: 1 px verticalOffset: 2 px.
 	insetShadow beInset.
 	shadow := CssBoxShadow
 		horizontalOffset: 1 px
 		verticalOffset: 2 px
 		blurRadius: 1 px
 		spreadDistance: 1 px
-		color: CssSVGColors white.
-	
+		color: CssConstants >> #colors >> #white.
```
Comment (fortizpenaloza) [inline source/RenoirSt-Tests/CssBoxShadowTest.class.st]: Are we loosing information changing SVGColors to just #colors ?

**Pull request merged**